.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,29 +0,0 @@
----
-name: 🐛 Bug report
-about: Tell us about a problem you are experiencing.
-title: ''
-labels: ''
-assignees: ''
-
----
-
-**What steps did you take and what happened:**
-[A clear and concise description on how to REPRODUCE the bug.]
-
-
-**What did you expect to happen:**
-
-
-**Anything else you would like to add:**
-[Miscellaneous information that will assist in solving the issue.]
-
-
-**Environment:**
-
-- Cluster-api version:
-- minikube/kind version:
-- Kubernetes version: (use `kubectl version`):
-- OS (e.g. from `/etc/os-release`):
-
-/kind bug
-[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]
```

Comment:
Is there a way to allow users to add area labels in the issue?

Reply:
Would preserving the same text at the end of the form work?

Reply:
@fabriziopandini area labels will be shown the same way as we had in soon be removed bug_report.md.